cli/flags.rs

```diff
@@ -107,6 +106,7 @@ pub struct Flags {
   pub ca_file: Option<String>,
   pub cached_only: bool,
   pub config_path: Option<String>,
+  pub cover: bool,
```

Comment:
@caspervonb one more nitpick, I think `coverage` is more meaningful than `cover`